doc/developer_guide/index.md

```diff
@@ -85,24 +105,38 @@ Syncing the git tags can be done with:
 pixi run sync-git-tags
 ```
 
-### Editable install
+## Developer Environment
 
-It can be advantageous to install the HoloViews in [editable mode](https://pip.pypa.io/en/stable/topics/local-project-installs/#editable-installs):
+The `default` environment is meant to provide all the tools needed to develop HoloViews.
 
-```bash
-pixi run install
-```
+This environment can be created by running `pixi run install`, which will set up the environment and make an editable install of HoloViews.
 
-:::{admonition} Note
-:class: info
+You can activate this environment in your terminal by running `pixi shell`.
 
-Currently, this needs to be run for each environment. So, if you want to install in the `test-ui` environment, you can add `--environment` / `-e` to the command:
+### VS Code
 
-```bash
-pixi run -e test-ui install
-```
+This environment can also be selected in your IDE. In VS Code, this can be done by running the command `Python: Select Interpreter` and choosing `{'default': Pixi}`.
 
-:::
+<p style="text-align: center">
+  <img
+    src="https://github.com/user-attachments/assets/6a8f7f60-870d-4497-8e23-52b336cf7682"
```

Comment:
Are you going to upload these files somewhere else or leave them on Github?

Reply:
Uploaded on S3